src/js/jwt/jwt.js

```diff
@@ -209,7 +210,11 @@ function logout(bounce) {
     cookie.remove(priv.cookie.cookieName);
 
     const isBeta = (window.location.pathname.split('/')[1] === 'beta' ? '/beta' : '');
-
+    const keySets = [
```

Comment:
You don't have to use two filters, just use one and we are good to go
```JS
const keys = Object.keys(localStorage).filter(key => (
  key.endsWith('/api/entitlements/v1/services') || key.endsWith('/config/main.yml')
))
```

And use them like
```JS
deleteLocalStorageItems(keys)
```